test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -1268,7 +1448,7 @@ def foo(
     "expected_circuit, ir",
     [
         (
-            Circuit().h(0, control=1, control_state=0),
+            Circuit().h(0, control=1, control_state=0).measure(0).measure(1),
```

Comment:
Just a note that by modifying these tests, we lose coverage on the case where the `measure` is implicit. Maybe that's fine if there's enough coverage of that case elsewhere - just wanted to mention it.

Reply:
These tests only cover `Circuit.from_ir` which now adds any use of measure in the OpenQASM as a `Measure` instruction on the circuit.

So `b[0] = measure q[0];` will now always be added to the circuit as `.measure(0)` when converting the OpenQASM ir to a circuit. 

The implicit measure is still covered in the `Circuit.to_ir` tests (ie. `Circuit.h(0)` still assumes qubit 0 is measured)

Reply:
Oh, sorry I completely misunderstood that. Thanks for clearing up my confusion.